assets/js/apps/search/components/filters/range-filter/NumberRangeFilter.jsx

```diff
@@ -0,0 +1,135 @@
+    if (!localValue) {
+        return {};
+    }
+    const submitValue = [];
+    if (!isNone(localValue.min)) {
+        submitValue.push({op:">=",content:localValue.min})
+    } 
+    if (!isNone(localValue.max)) {
+        submitValue.push({op:"<=",content:localValue.max});
+    }
+    if (submitValue.length === 0){
+        return null;
+    }
+    return submitValue;
+}
+
+const toLocalValue = submitValue => {
+    // Replace null value with empty string to represent null parameter value.
+    if (!submitValue) {
+        return {};
+    }
+    if (!Array.isArray(submitValue)) {
+        // Wrap value in array if not already in array.
+        submitValue = [submitValue];
+    }
+    const localValue = {};
+    // Iterate over the filter values to get the min and max values.
+    submitValue.forEach(filter => {
+        if (!filter || !typeof filter === "object" || isNone(filter.op) || isNone(filter.content)) {
+            return;
+        }
+        if (filter.op === ">=") {
+            localValue.min = filter.content;
+        }
+        if (filter.op === "<=") {
+            localValue.max = filter.content;
+        }
+    });
+    return localValue;
+}
+
+const NumberRangeFilter = ({value,options,onValueChange}) => {
+    // Make a copy of the options first.
+    options = Object.assign({},options);
+    if (!options.name){
+        options.name = "Missing filter name";
+    }
+    if (!options.hint) {
+        options.hint = "";
+    }
+    const [localValue, setLocalValue] = useState(toLocalValue(value));
+    useEffect(() => {
+        setLocalValue(toLocalValue(value));
+    }, [value]);
+    const handleValueChange = (type,e) => {
+        // Copy the value object, then assign new value into either "min" or "max".
+        const valueFromForm = e.target.value;
+        const newValue = Object.assign({},localValue);
+        newValue[type] = valueFromForm;
+        setLocalValue(newValue);
+    };
+
+    // We should disable the filter button if there's nothing in the filter box.
+    // But we should be able to clear a field if there's a value on the filter.
+    const canChangeValue =  !isValueEmpty(localValue) || !isValueEmpty(toLocalValue(value));
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if (!canChangeValue) {
+            return;
+        }
+        onValueChange(toSubmitValue(localValue));
+    };
+    return (
+        <Form className="num-range-filter" onSubmit={handleSubmit}>
+                    <Form.Group className="num-range-filter__field">
+                        <Form.Label>Min</Form.Label>
+                        <Form.Control  
+                            onChange={handleValueChange.bind(this,"min")} 
+                            value={localValue.min} 
+                            aria-label="Filter input for min value"
+                            placeholder={options.hintMin}
+                        >
+                        </Form.Control>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 16 spaces but found 24.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
                </Form.Control>
```